cfgen/cfgen

```diff
@@ -1854,7 +1858,38 @@ def build_cluster(cluster_desc: Dict[str, Any]) -> Cluster:
                 proc_id = ConfProcess.build(conf, node_id, node, proc_t)
                 cluster.m0_clients[proc_id] = new_oid(ObjT.service)
 
+    aux_pools = {}
+    disk_ref_combinations = []
     for pool in cluster_desc['pools']:
+        parity_group_size = pool['data_units'] + pool['parity_units'] +\
+                            pool['spare_units']
+        if len(pool.get('disk_refs', [])) == parity_group_size:
```

Comment:
@mssawant 
This is that strict condition of disks to be equal.
Once you check code / conditions and confirm, I will remove this.
Thanks!

Reply:
Removed.
Algorithm is now completely changed from the original one.
Original algo was creating combinations considering tolerance values of disks.
New algo creates node combinations considering tolerance values for node first and then creates lists of corresponding disks, accordingly.